modules/oe_theme_content_event/src/Plugin/ExtraField/Display/MediaExtraField.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_event\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\oe_content_event\EventNodeWrapper;
+
+/**
+ * Extra field displaying either the event media related fields.
+ *
+ * @ExtraFieldDisplay(
+ *   id = "oe_theme_content_event_media",
+ *   label = @Translation("Media gallery"),
+ *   bundles = {
+ *     "node.oe_event",
+ *   },
+ *   visible = true
+ * )
+ */
+class MediaExtraField extends DateAwareExtraFieldBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getLabel() {
+    return $this->t('Media gallery');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(ContentEntityInterface $entity) {
+    $event = new EventNodeWrapper($entity);
+    $build = [
+      '#theme' => 'oe_theme_content_event_media',
+      '#items' => [],
+    ];
+
+    // If the event is not over then apply time-based tags, so that it can be
+    // correctly invalidated once the event is over.
+    if (!$event->isOver($this->requestDateTime)) {
+      $this->applyHourTag($build, $event->getEndDate());
+    }
+
+    /** @var \Drupal\Core\Entity\EntityViewBuilderInterface $view_builder */
+    $view_builder = $this->entityTypeManager->getViewBuilder('node');
+    if (!$event->isOver($this->requestDateTime)) {
+      $this->isEmpty = TRUE;
+      return $build;
+    }
+    if (!$entity->get('oe_event_media')->isEmpty()) {
+      $build['#items']['media_gallery'] = $view_builder->viewField($entity->get('oe_event_media'), [
+        'label' => 'hidden',
+        'type' => 'oe_theme_helper_media_gallery',
+      ]);
+    }
+    if (!$entity->get('oe_event_media_more_link')->isEmpty()) {
+      $build['#items']['media_more_link'] = [
+        'url' => $entity->get('oe_event_media_more_link')->getValue()[0]['uri'],
+        'title' => $entity->get('oe_event_media_more_link')->getValue()[0]['title'],
+      ];
+    }
+    if (!$entity->get('oe_event_media_more_description')->isEmpty()) {
+      $build['#items']['media_more_description'] = $view_builder->viewField($entity->get('oe_event_media_more_description'), [
+        'label' => 'hidden',
+      ]);
+    }
+    if (count($build['#items']) < 1) {
+      $this->isEmpty = TRUE;
```

Comment:
![image](https://user-images.githubusercontent.com/22004498/147928245-6346edea-146a-41d3-a9a1-ab342e079824.png)
I didn't fill any of these fields in.

Reply:
When I fill it in, I get double titles.
![image](https://user-images.githubusercontent.com/22004498/147928483-ca29417f-7e1c-40b6-9fe1-e9320908654d.png)